element_array_ephys/ephys_acute.py

```diff
@@ -229,113 +263,147 @@ class EphysFile(dj.Part):
         """
 
     def make(self, key):
-        session_dir = find_full_path(get_ephys_root_data_dir(),
-                                  get_session_directory(key))
-        inserted_probe_serial_number = (ProbeInsertion * probe.Probe & key).fetch1('probe')
+        session_dir = find_full_path(
+            get_ephys_root_data_dir(), get_session_directory(key)
+        )
+        inserted_probe_serial_number = (ProbeInsertion * probe.Probe & key).fetch1(
+            "probe"
+        )
 
         # search session dir and determine acquisition software
-        for ephys_pattern, ephys_acq_type in zip(['*.ap.meta', '*.oebin'],
-                                                 ['SpikeGLX', 'Open Ephys']):
+        for ephys_pattern, ephys_acq_type in zip(
+            ["*.ap.meta", "*.oebin"], ["SpikeGLX", "Open Ephys"]
```

Comment:
Hm.. this pattern again. It's actually clearer without the `zip`.
```suggestion
        for ephys_pattern, ephys_acq_type in (
            ("*.ap.meta", "SpikeGLX"), ("*.oebin", "Open Ephys")
```